mods/cnc/chrome/settings.yaml

```diff
@@ -336,7 +336,7 @@ Container@SETTINGS_PANEL:
 							Width: 300
 							Height: 25
 						Label@AUDIO_DEVICE_DESC:
-							Y: 265
+							Y: 264
```

Comment:
This is a couple of px too high compared to the previous release and compared to what IMO looks good.